### PR TITLE
add stompy mini to sim

### PR DESCRIPTION
**Pull request by wuallenwu**

### CODE REVIEW

File: source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy_mini.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
```
Comment (budzianowski) [inline source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy_mini.py:1]:
```suggestion
# Copyright (c) 2024 KScale Labs.
```

File: source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy_mini.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Configuration for Stompy."""
+
+import omni.isaac.lab.sim as sim_utils
+from omni.isaac.lab.actuators import ImplicitActuatorCfg
+from omni.isaac.lab.assets.articulation import ArticulationCfg
+
+STOMPYMINI_CFG = ArticulationCfg(
+    # Spawn stompy from URDF
```
Comment (budzianowski) [inline source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy_mini.py:13]: remove

File: source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/stompy_mini/__init__.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
```
Comment (budzianowski) [inline source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/stompy_mini/__init__.py:1]:
```suggestion
# Copyright (c) 2024 KScale Labs.
```

File: source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/stompy_mini/flat_env_cfg.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        super().__post_init__()
+
+        # make a smaller scene for play
+        self.scene.num_envs = 50
+        self.scene.env_spacing = 2.5
+        # disable randomization for play
+        self.observations.policy.enable_corruption = False
+        # remove random pushing
+        self.events.base_external_force_torque = None
+        self.events.push_robot = None
```
Comment (budzianowski) [inline source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/stompy_mini/flat_env_cfg.py:41]: Are these default values for H1 as well?

**Review by budzianowski — APPROVED** [review]
Small nits, remove the pt file, otherwise lgtm.

File: source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy_mini.py (hunk 2 above)
Comment (wuallenwu) [reply source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy_mini.py]: fixed

File: source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/stompy_mini/flat_env_cfg.py (hunk 4 above)
Comment (wuallenwu) [reply source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/stompy_mini/flat_env_cfg.py:41]: yes, i made sure all of the hyperparameters match the H1

**Pull request merged**